src/NuGet.Core/NuGet.DependencyResolver.Core/Strings.resx

```diff
@@ -121,11 +121,11 @@
     <value>The feed '{0}' lists package '{1}' but multiple attempts to download the nupkg have failed. The feed is either invalid or required packages were removed while the current operation was in progress. Verify the package exists on the feed and try again.</value>
   </data>
   <data name="Log_MatchingSourceFoundForPackage" xml:space="preserve">
-    <value>Package namespace matches found for package ID '{0}' are: '{1}'.</value>
+    <value>Package source mapping pattern matches found for package ID '{0}' are: '{1}'.</value>
```

Comment:
You might've already done it, but I'd rather provide the feedback than risk us missing something. 

Consider search for the word `namespace` in all resx files in our repo.

My guess is that we shouldn't have any hits.

Reply:
I did search for `namespace` in all the files in the repo (that includes resx files) and could not find a keyword `namespaces` in the context of `source mapping` feature.

Reply:
Awesome! Thanks for being so thorough! Very important with a PR like this.